packages/m2m-gateway/src/services/eserviceService.ts

```diff
@@ -270,6 +270,35 @@ export function eserviceServiceBuilder(
         throw eserviceDescriptorNotFound(eServiceId, descriptorId);
       }
 
+      return toM2MGatewayApiEServiceDescriptor(descriptor);
+    },
+    async approveDelegatedEServiceDescriptor(
+      eServiceId: EServiceId,
```

Comment:
Consistency in this file
```suggestion
      eserviceId: EServiceId,
```